Source/carthage/Build.swift

```diff
@@ -128,6 +132,17 @@ public struct BuildCommand: CommandType {
 			return buildProducer
 		} else {
 			let currentProducers = buildInDirectory(directoryURL, withConfiguration: options.configuration, platform: options.buildPlatform.platform)
+				|> catch { error -> SignalProducer<BuildSchemeProducer, CarthageError> in
+					switch error {
+					case let .NoSharedFrameworkSchemes(project, _):
+						// Log that building the current project is being skipped.
+						eventSink.put(.SkippedBuilding(project, error.description))
+						return .empty
+
+					default:
+						return SignalProducer(error: error)
+					}
+				}
```

Comment:
Since the error case is repacked in `buildDependencyProject()` for building each dependencies, catching it in `buildInDirectory()` level is not the idea, I think. Additionally `buildInDirectory()` does not know about `Project` nor `ProjectEventSink`, so it seems difficult to send the event from the function atm. Do you have any idea to fix it?